pkg/runtime/meta/meta.go

```diff
@@ -96,3 +99,12 @@ func ContainsNamespace(items []common.NameValuePair) bool {
 	}
 	return false
 }
+
+// AddWasmStrictSandbox adds wasm strict sandbox configuration to metadata.
+func AddWasmStrictSandbox(comp *compapi.Component, enable bool) {
```

Comment:
Done.
1. if `enable` is undefined, do nothing, keep the user's configuration. Otherwise:
2. scan `comp.Spec.Metadata` if `WasmStrictSandboxMetadataKey` exists update, else add.